tensorflow/lite/CMakeLists.txt

```diff
@@ -98,6 +98,8 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
   # 2b96f3662bd776e277f86997659e61046b56c315/tensorflow/lite/tools/make/\
   # Makefile#L157
   set(_TFLITE_ENABLE_MMAP OFF)
+  #Should this also be true if TFLITE_ENABLE_MMAP=OFF even if windows is not the system name?
```

Comment:
Given that when the system is windows, MMAP already set to off, is this compile option necessary?